tools/Dashboard/DevHome.Dashboard/Services/IWidgetScreenshotService.cs

```diff
@@ -10,5 +10,7 @@ namespace DevHome.Dashboard.Services;
 
 public interface IWidgetScreenshotService
 {
-    public Task<BitmapImage> GetScreenshotFromCache(WidgetDefinition widgetDefinition, ElementTheme actualTheme);
+    public void RemoveScreenshotsFromCache(string definitionId);

```

Comment:
Remove `cache` from the method names.  To me, if I see a method with `cache` I expect other methods that pull from different locations to be included in the interface. RemoveScreenshotsFrom`File` maybe.  However, that introduces the question of "What location should I remove from, `File` or `cache`.

Additionally, adding `cache` somewhat forces implementers to have a cache.  (Technically each object, usually, has a cache).

Reply:
#2482